libs/app-utils/main/src/lib/app-help/app-help.service.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import path from "path"
+import { pathExists, readJSON, writeJSON } from "fs-extra"
+import axios from "axios"
+import defaultHelp from "./default-help.json"
+import { HelpData } from "./app-help.types"
+import { BrowserWindow } from "electron"
+import { HelpIpcEvents } from "app-utils/models"
+
+const helpPath = path.join(process.cwd(), "help-v2.json")
+
+export class AppHelpService {
+  private mainWindow: BrowserWindow | null = null
+
+  setMainWindow(window: BrowserWindow) {
+    this.mainWindow = window
+  }
+
+  async initialize(): Promise<void> {
+    const helpExists = await pathExists(helpPath)
+    if (!helpExists) {
+      const freshHelp = await this.getNewestData()
+      const helpToSave = freshHelp ?? defaultHelp
+      await writeJSON(helpPath, helpToSave)
```

Comment:
Please refer to https://github.com/mudita/mudita-center/blob/CP-3622/libs/news/main/src/lib/news.service.ts and use the `jsonStore`